src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,122 @@
+        List<Manufacturer> manufacturers = new ArrayList<>();
+        String query = "SELECT * FROM manufacturers WHERE is_deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement getAllManufacturers = connection.createStatement()) {
+            ResultSet resultSet = getAllManufacturers.executeQuery(query);
+            while (resultSet.next()) {
+                Manufacturer manufacturer = setManufacturerFromResultSet(resultSet);
+                manufacturers.add(manufacturer);
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't get all manufacturers from DB", throwables);
+        }
+        return manufacturers;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String insertFormatRequest =
+                "UPDATE manufacturers SET name = ?, "
+                        + "country = ? WHERE (id = ? AND is_deleted = false);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturerStatement = connection
+                        .prepareStatement(insertFormatRequest)) {
+            createManufacturerStatement.setString(1, manufacturer.getName());
+            createManufacturerStatement.setString(2, manufacturer.getCountry());
+            createManufacturerStatement.setObject(3, manufacturer.getId());
+            if (createManufacturerStatement.executeUpdate() >= 1) {
+                return manufacturer;
+            }
+            throw new RuntimeException("There is no such manufacturer");
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't update manufacturer with id: "
+                    + manufacturer.getId() + " to DB", throwables);
+        }
+    }
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertFormatRequest =
+                "INSERT INTO manufacturers(name, country) VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturerStatement = connection
+                        .prepareStatement(insertFormatRequest, Statement.RETURN_GENERATED_KEYS)) {
+            createManufacturerStatement.setString(1, manufacturer.getName());
+            createManufacturerStatement.setString(2, manufacturer.getCountry());
+            createManufacturerStatement.executeUpdate();
+            ResultSet generatedKeys = createManufacturerStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't insert manufacturer with id: "
+                    + manufacturer.getId() + " to DB", throwables);
```

Comment:
```suggestion
                    + manufacturer + " to DB", throwables);
```
If you didn't insert it, manufacturer has no id